hat/assets/js/apps/Iaso/domains/periods/models.js

```diff
@@ -84,6 +93,7 @@ export class Period {
     }
 
     static parse(periodString) {
+        if (!periodString) return null;
```

Comment:
i had a reason of doing the` throw error` at the end i think

Reply:
I didn't remove it. It's just that if you throw on `null` the component will crash when no date has been selected